deps/rabbitmq_management_agent/src/rabbit_mgmt_external_stats.erl

```diff
@@ -108,25 +108,23 @@ get_used_fd({win32, _}, State0) ->
     case os:find_executable("handle.exe") of
         false ->
             State1 = log_fd_warning_once("Could not find handle.exe, using powershell to determine handle count", [], State0),
```

Comment:
This log message is no longer correct, is it?

Reply:
Yessir, I sure missed that one.